src/components/common/Settings.tsx

```diff
@@ -13,6 +9,10 @@ import Switch from '@mui/material/Switch';
 import Tooltip from '@mui/material/Tooltip';
 import Typography from '@mui/material/Typography';
 
+import { AppSetting } from '@graasp/sdk';
+
+import { makeStyles } from 'tss-react/mui';
```

Comment:
If you think of updating the file drop further (not in this PR) you can slowly remove those `makeStyles` to using `sx` or styled components.